domain/todo_usecase_interface.go

```diff
@@ -2,8 +2,11 @@ package domain
 
 import (
 	"context"
+	"github.com/google/uuid"
 )
 
 type TodoUseCase interface {
 	CreateTodoUseCase(ctx context.Context, entity CreateTodoInputEntity) (result *CreateTodoEntity, err error)
+	GetAllTodoUseCase(ctx context.Context) (result []GetTodoEntity, err error)
+	GetTodoUseCase(ctx context.Context, key uuid.UUID) (result []GetTodoEntity, err error)
```

Comment:
Can we group GetAllTodoUseCase and GetTodoIUseCase ?
If not why?

Reply:
It is possible to set method name as 
- CreateTodo(ctx context.Context, entity CreateTodoInputEntity) (result *CreateTodoEntity, err error)
- GetAllTodo(ctx context.Context) (result []GetTodoEntity, err error)
- GetTodo(ctx context.Context, key uuid.UUID) (result []GetTodoEntity, err error)

Reply:
at first i think it's should be separate but i'm forgot it can be in 1 function